iocBoot/iocCamera/post-init.cmd

```diff
@@ -1,4 +1,21 @@
 # Post IOC initialization setup
+#
+# Optional parameters:
+#
+# $(USE_COMPRESSION)
+# Use recommended compression settings (Yes or No). Defaults to No.
 
 create_monitor_set("ioc.req", 30, "P=$(PREFIX)")
 set_savefile_name("ioc.req", "$(PREFIX).sav")
+
+epicsEnvSet("USE_COMPRESSION", "$(USE_COMPRESSION=No)")
+
+epicsEnvSet("COMPRESSOR_No", "None")
+epicsEnvSet("COMPRESSOR_Yes", "BSLZ4")
+epicsEnvSet("COMPRESSION_THREADS_No", 1)
+epicsEnvSet("COMPRESSION_THREADS_Yes", $(MAX_THREADS))
+
+dbpf $(PREFIX)Codec1:EnableCallbacks Enable
+dbpf $(PREFIX)Codec1:Mode Compress
+dbpf $(PREFIX)Codec1:Compressor $(COMPRESSOR_$(USE_COMPRESSION))
+dbpf $(PREFIX)Codec1:NumThreads $(COMPRESSION_THREADS_$(USE_COMPRESSION))
```

Comment:
Should we do any writes into `SortSize` as well? https://areadetector.github.io/areaDetector/ADCore/NDPluginDriver.html#sorting-of-output-ndarrays